bombbot.py

```diff
@@ -212,8 +212,12 @@ def cancel_bomb(bot, trigger):
     """
     target = trigger.group(3) or None
     if not target:
```

Comment:
I would _love_ to convert this into a `while` loop, but then I'd be worried about iterating over `BOMBS` without the syntactic sugar of `for ... in ...:` to help avoid threading issues.

That said, because the other code that modifies `BOMBS` does so in-place, perhaps this iterator should run `with lock:` anyway, just in case.